src/features/CartPage/slices/cartSlice.js

```diff
@@ -0,0 +1,42 @@
+import {createSlice} from '@reduxjs/toolkit';
+
+const initialState = {
+  cart: [],
+};
+
+export const cartSlice = createSlice({
+  name: 'myCart',
+  initialState,
+  reducers: {
+    setCart: (state, {payload}) => {
+      state.cart = payload;
+      localStorage.setItem('cart', JSON.stringify(state.cart));
```

Comment:
is storing cart data in localStorage demanded by the assignment? if no, then what would be the concise motivation for doing so?

Reply:
For guest user (not signed in)